android/kotlin_guide.md

```diff
@@ -0,0 +1,49 @@
+# Kotlin guide
+This guide is designed to help you get started with [Kotlin], the language we write most of our code in, and to introduce you to some intermediate topics.
+- [Getting Started](#getting-started)
+- [Staying Informed](#staying-informed)
+- [Intermediate Topics](#intermediate-topics)
+
+## Getting Started
+There are multiple ways to get started with Kotlin, depending on how you like to learn. If you like:
+- Reading documentation, the official Kotlin docs are good: start at [Basic Syntax] and work your way through the pages in the side bar
+- Learning while coding, try the [Kotlin Koans] online or in the IDE
```

Comment:
Might be good to briefly describe what the Koans are.  

Also IMO, if someone has no Kotlin experience and needs to ramp up quickly, Koans should be their first step.  If there's agreement on that point we might want to mention it here.